tests/e2e/specs/info.cy.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+describe('Info View', () => {
+  it('works', () => {
+    // test opening the "Info View" from a task in the "Tree View"
+    cy.visit('/#/workspace/one')
+      // click on task 20000102T0000Z/failed
+      .get('.c-treeitem .c-treeitem .c-treeitem:first')
+      .find('.c-task')
+      .click({ force: true })
+
+      // from the menu select the "Info" psudo-mutation
+      .get('.v-list > :nth-child(6)')
```

Comment:
Can make these less brittle
```suggestion
      // click on task
      .get('.node-data-task [data-c-interactive]:first')
      .click({ force: true })

      // from the menu select the "Info" psudo-mutation
      .get('.v-list-item')
```